app/src/main/java/com/sunasterisk/boringweather/data/Repository.kt

```diff
@@ -0,0 +1,4 @@
+package com.sunasterisk.boringweather.data
+
+interface Repository {
```

Comment:
```suggestion
interface RepositoryType {
```
để tên theo từng màn hình nhé. Ví dụ: 
`interface HomeRepositoryType`
 và dùng như này
`class HomeRepository: HomeRepository`